### DISCUSSION TITLE
Javascript, Typescript Files are not indented with 2 spaces

### DISCUSSION THREAD

**Discussion by scimania1** [nvim-treesitter/nvim-treesitter#3999] [Q&A]
Indentation of Javascript and Typescript files is still 4 spaces even after setting `indent = {enable = true}` in the config for treesitter. Could anyone help me through this problem?

**Comment by theHamsta** [comment]
You still have the manually set `:h 'shiftwidth'` and other options regarding indentation. The indent module only decides when to indent, not how (space vs tabs, how many spaces, etc..)
